code/game/objects/effects/anomalies.dm

```diff
@@ -276,11 +276,17 @@
 	S.rabid = TRUE
 	S.amount_grown = SLIME_EVOLUTION_THRESHOLD
 	S.Evolve()
+	var/datum/action/innate/slime/reproduce/A = new
+	A.Grant(S)
 
-	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a pyroclastic anomaly slime?", ROLE_PAI, null, null, 100, S, POLL_IGNORE_PYROSLIME)
+	var/list/mob/dead/observer/candidates = pollCandidatesForMob("Do you want to play as a pyroclastic anomaly slime?", ROLE_SENTIENCE, null, null, 100, S, POLL_IGNORE_PYROSLIME)
```

Comment:
shouldnt this have the new role you are adding

Reply:
@Fikou  ` ROLE_SENTIENCE` is the main role.  It is what you have to check in character settings if you would like to play anomaly slime and it is what  admemes will ban to prevent you from playing it.  ROLE_PYROCLASTIC_SLIME is a `special_role` that if added as a prime role will clutter stuff and make admemes remember that you also have to ban it. It probably means modifying the banning template and stuff. The role exists just to add additional policy on top of the ` ROLE_SENTIENCE` has.

Reply:
thats dumb